internal/db/schema/migrations/oss/postgres/19/03_db.up.sql

```diff
@@ -0,0 +1,23 @@
+begin;
```

Comment:
Does this file need to be in `20/` or is it fine to append to the latest migration major number so long as it's strictly increasing?

Reply:
I confirmed with the team, that major versions can include additional migrations for new PRs as long as the major version hasn't be included within a Boundary release.